bin/load-spark-env.cmd

```diff
@@ -29,10 +29,7 @@ if [%SPARK_ENV_LOADED%] == [] (
     set SPARK_CONF_DIR=%~dp0..\conf
   )
 
-  set SPARK_ENV_CMD=%SPARK_CONF_DIR%\%SPARK_ENV_CMD%
-  if exist %SPARK_ENV_CMD% (
-    call %SPARK_ENV_CMD%
-  )
+  call :LoadSparkEnv
```

Comment:
This seems to be changed via SPARK-26132 (https://github.com/apache/spark/commit/8bc304f97ee693b57f33fa6708eb63e2d641c609).

Reply:
As I understand, inside cmd, set env twice not working. So `set SPARK_ENV_CMD=%SPARK_CONF_DIR%\%SPARK_ENV_CMD%` doesn't take effect, which caused regression.

Reply:
Could you put that explanation into `PR description`? PR title and description will be a permanent commit log.

Reply:
Also, please describe how did you do for `Manually tested`.

Reply:
Updated.

Reply:
Yeah .. I vaguely remember setting env twice isn't allowed ..

Reply:
I have confirmed this issue is env set inside `IF` won't take effect until after whole block executed. Please refer https://superuser.com/questions/78496/variables-in-batch-file-not-being-set-when-inside-if